next.config.mjs

```diff
@@ -23,7 +23,7 @@ const babelIncludeRegexes = [
  * @type {import('next').NextConfig}
  * */
 const nextConfig = {
-  reactStrictMode: true,
+  reactStrictMode: process.env.NODE_ENV !== 'development',
   compiler: {
```

Comment:
Did you turn off strict mode because it renders components twice?

Reply:
@storywithoutend yes it's to prevent the useEffect running twice